shared/src/database/services/donation-certificate/donation-certificate-writer.ts

```
@@ -0,0 +1,162 @@
+			pdfDocument.text(footerLeftLine3, 45, pdfDocument.page.height - 40, { lineBreak: false });
+			pdfDocument.text(footerMiddleLine1, 215, pdfDocument.page.height - 70, { lineBreak: false });
+			pdfDocument.text(footerMiddleLine2, 215, pdfDocument.page.height - 55, { lineBreak: false });
+			pdfDocument.text(footerMiddleLine3, 215, pdfDocument.page.height - 40, { lineBreak: false });
+			pdfDocument.text(footerRightLine1, 405, pdfDocument.page.height - 70, { lineBreak: false });
+			pdfDocument.text(footerRightLine2, 405, pdfDocument.page.height - 55, { lineBreak: false });
+			pdfDocument.text(footerRightLine3, 405, pdfDocument.page.height - 40, { lineBreak: false });
+
+			pdfDocument.pipe(writeStream);
+			writeStream.on('finish', () => resolve());
+			pdfDocument.end();
+		});
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)